must-gather/collection-scripts/gather_crds

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+source pwait
+max_parallelism=10
+
+# Cluster passed in from main gather
+clusterID=$1
+
+# Resource list
+resources=()
+
+# OADP
+resources+=($(/usr/bin/oc get crd | awk '/oadp.openshift.io/{print $1}'))
+
+# Velero
+resources+=($(/usr/bin/oc get crd | awk '/velero.io/{print $1}'))
+
+echo "Starting collection of CRDs: [${resources[@]}]"
+object_collection_path=/must-gather/clusters/${clusterID}/cluster-scoped-resources/apiextensions.k8s.io/customresourcedefinitions
+
+for resource in ${resources[@]}; do
+  echo "Collecting ${resource} CRD"
+  mkdir -p ${object_collection_path}
+  /usr/bin/oc get crd ${resource} -o yaml &> ${object_collection_path}/${resource}.yaml &
```

Comment:
Should that be also surrounded by quotes, we don't know if var names have spaces in them or not, especially it's path.